bool use_defaults
bool delete_prev_traj
float32 x
float32 y
float32 z
float32 direction_x
float32 direction_y
float32 direction_z
float32 search_distance
---
bool success
string message
